pkg/apis/provisioning/v1alpha5/register.go

```diff
@@ -57,7 +57,6 @@ var (
 		v1.LabelTopologyZone,
 		v1.LabelInstanceTypeStable,
 		v1.LabelArchStable,
-		v1.LabelOSStable,
```

Comment:
This is still a well known label that we will support for Windows eventually.

Reply:
This currently will break scheduling assumptions since the options are [] and it will be overly constrained. Let's add it back in when we support it.